addons/stock/tests/test_move2.py

```diff
@@ -13,6 +13,46 @@
 from dateutil.relativedelta import relativedelta
 
 class TestPickShip(TestStockCommon):
+
+    def setUp(self):
```

Comment:
Be careful that a setUp function is executed before **each** test methods of the class. As I don't see any interest of creating the same product multiple time, I would create it directly in the test and remove the `setUp()`

Reply:
Done! Sorry but i can't find any complete guidelines about the odoo tests . I learned reading the code of other functions ;)

Reply:
I have already signed the CLA in branch 16.0 https://github.com/odoo/odoo/blob/16.0/doc/cla/corporate/stesiconsultingsrl.md but there is the CLA review requested here